Sources/ProjectDescription/Cache.swift

```diff
@@ -16,6 +16,8 @@ public struct Cache: Codable, Equatable {
         /// The version of the OS to be used when building the project during a caching warmup
         public let os: String?
 
+        public let deviceOnlyXCFramework: Bool
```

Comment:
^ I agree with Dan's suggestion about being able to specify device/sim for --xcframeworks

Reply:
Thanks for suggestion, I'll take a look, I think it definitely possible, but it should be another PR.

Reply:
This might has some problem with cache invalidation because it produces different xcframeworks but currently tuist only compute hashes from source files/cache profile/output type, I think?

@danyf90 @kwridan @fortmarek

Reply:
Hmm I think it shouldn't be a problem if I just split cache output type of xcframework into 2 types. But in main branch, ./fourier generate tuist no longer generate xcworkspace anymore, any know why? 😁

Reply:
> Thanks for suggestion, I'll take a look, I think it definitely possible, but it should be another PR.

I would make this part of this PR, I don't see the point of merging one approach to then change it

Reply:
I mean I'm gonna close this. I'm ready to push another PR with suggested approach, because there are conflicts between these.